packages/desktop-client/src/components/manager/BudgetList.jsx

```diff
@@ -343,7 +351,9 @@ export function BudgetList() {
             type="primary"
             isSubmit={false}
             onClick={() => onCreate({ testMode: true })}
-            style={{ marginLeft: 15 }}
+            style={{
+              ...narrowButtonStyle,
+            }}
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏
```suggestion
            style={narrowButtonStyle}
```